metapackages/auto-instrumentations-node/src/utils.ts

```diff
@@ -149,6 +149,12 @@ export function getNodeAutoInstrumentations(
   const enabledInstrumentationsFromEnv = getEnabledInstrumentationsFromEnv();
   const disabledInstrumentationsFromEnv = getDisabledInstrumentationsFromEnv();
 
+  if (enabledInstrumentationsFromEnv && disabledInstrumentationsFromEnv) {
+    diag.debug(
+      'OTEL_NODE_ENABLED_INSTRUMENTATIONS and OTEL_NODE_DISABLED_INSTRUMENTATIONS environment variables are mutually exclusive. All instrumentations are disabled.'
+    );
+  }
```

Comment:
But this isn't the actual behaviour, is it?

```
% cd ~/src/opentelemetry-js-contrib/metapackages/auto-instrumentations-node
% node
> var mod = require('./')
undefined
> process.env.OTEL_NODE_ENABLED_INSTRUMENTATIONS = 'dns,fs,hapi'
'dns,fs,hapi'
> mod.getNodeAutoInstrumentations().map(instr => instr.instrumentationName)
[
  '@opentelemetry/instrumentation-dns',
  '@opentelemetry/instrumentation-fs',
  '@opentelemetry/instrumentation-hapi'
]
> process.env.OTEL_NODE_DISABLED_INSTRUMENTATIONS = 'bunyan,fs'
'bunyan,fs'
> mod.getNodeAutoInstrumentations().map(instr => instr.instrumentationName)
[
  '@opentelemetry/instrumentation-dns',
  '@opentelemetry/instrumentation-hapi'
]
```

(See related comment at https://github.com/open-telemetry/opentelemetry.io/pull/4824#discussion_r1680436482

Reply:
Huh yeah that was a poor test, not sure what I was doing 😅 . Will update!